cli/src/pcluster/models/cluster.py

```diff
@@ -695,6 +696,29 @@ def stop(self):
         except Exception as e:
             raise _cluster_error_mapper(e, f"Failed when stopping compute fleet with error: {str(e)}")
 
+    def stop_slurm_compute_fleet(self):
```

Comment:
The stop function was split to two to solve `function too complex` complain by linters